inst/JOSS/paper.Rmd

```diff
@@ -187,18 +193,33 @@ ha_record <- bcdc_get_record("7bc6018f-bb4f-4e5d-845e-c529e3d1ac3b")
 ## Have a quick look at the geospatial columns to help with filter or select
 bcdc_describe_feature(ha_record)
 
+## Naively download the whole data object, then filter it
+system.time(
+  all_ha <- bcdc_get_data(ha_record, resource = "93b79a3c-2da4-4fd4-b953-2f5c690db430")
+)
+object.size(all_ha)
```

Comment:
Would you consider `format(object.size(all_ha), units = "Mb")` here?

Reply:
i think the readability in output probably outweighs the obtuseness of the code ;)